pkg/segment/structs/segmeta.go

```diff
@@ -24,8 +24,9 @@ import (
 const MAX_SEGMETA_FSIZE = 10_000_000 // 10 MB
 
 type ColSizeInfo struct {
-	CmiSize uint64 `json:"cmiSize"`
-	CsgSize uint64 `json:"csgSize"`
+	CmiSize  uint64 `json:"cmiSize"`
+	CsgSize  uint64 `json:"csgSize"`
+	CValSize int8   `json:"cValSize"`
```

Comment:
Since this is just meta and will get store once in segmeta per segment, then we can just go with a type of `int` , that should handle the cases where the string length is greater than 127 and we don't have to worry about overflows.
Plus it is not of big hit of 1 byte v/s 4 bytes for this var type